gpflow/utilities/ops.py

```diff
@@ -102,6 +102,17 @@ def square_distance(X, X2):
     return dist
 
 
+def distance(X, X2):
```

Comment:
I would expect a "distance" to be scalar. Would it be more accurate to call this, e.g., `difference_matrix`?